src/midgard/gridded_data.cc

```diff
@@ -1,330 +0,0 @@
-#include "midgard/gridded_data.h"
```

Comment:
this was all moved to the header file. the bulk of this class is a single function